GitDiffMargin.Unit.Tests/Git/HunkRangeInfoTests.cs

```diff
@@ -38,78 +39,91 @@ public void IsAddition_AllDiffLinesWithStartsWithPlusSign_ExpectedTrue()
         public void IsAddition_NotAllDiffLinesStartsWithPlusSign_ExpectedFalse()
         {
             //Arrange
-            var hunkRangeInfo = new HunkRangeInfo(new HunkRange("-41,0", _contextLines), new HunkRange("+42,20", _contextLines), new List<string> { "+ ", "- " }.ToArray());
+            var hunkRangeInfo = new HunkRangeInfo(new HunkRange("-41,0", _contextLines),
+                new HunkRange("+42,20", _contextLines), new List<string> {"+ ", "- "}.ToArray());
 
             //Act
-            bool isAddition = hunkRangeInfo.IsAddition;
+            var isAddition = hunkRangeInfo.IsAddition;
 
             //Assert
             isAddition.ShouldBe(false);
         }
 
         [Test]
-        public void IsModification_DiffLinesStartsWithPlusSignAndWithMinus_ExpectedTrue()
+        public void IsDeletion_3DeletedLines_ExpectTrue()
         {
             //Arrange
-            var hunkRangeInfo = new HunkRangeInfo(new HunkRange("-41,0", _contextLines), new HunkRange("+42,20", _contextLines), new List<string> { "+ ", "- " }.ToArray());
+            var hunkRangeInfo = new HunkRangeInfo(new HunkRange("-7,3", _contextLines),
+                new HunkRange("+6,0", _contextLines),
+                new List<string>
+                {
+                    "-using Microsoft.VisualStudio.Shell;",
+                    "-using Microsoft.VisualStudio.Text;",
+                    "-using Microsoft.VisualStudio.Text.Editor;"
+                }.ToArray());
 
             //Act
-            bool isModification = hunkRangeInfo.IsModification;
+            var isDeletion = hunkRangeInfo.IsDeletion;

```

Comment:
:memo: This change is much less obvious until the reader realizes that the test methods were reordered for some reason. Reordering methods is a relatively difficult change to review, and should probably not be combined with other changes.